service/collector/elb.go

```diff
@@ -0,0 +1,136 @@
+package collector
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/giantswarm/aws-operator/client/aws"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	AccountLabel = "account"
+)
+
+var (
+	elbsDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "elb_instance_out_of_service_count"),
```

Comment:
Not sure if the name is good.

Reply:
Good enough for now. We don't have guidelines for metric naming yet.